api/envoy/config/listener/v3/listener.proto

```diff
@@ -358,3 +362,8 @@ message Listener {
   // :ref:`global_downstream_max_connections <config_overload_manager_limiting_connections>`.
   bool ignore_global_conn_limit = 31;
 }
+
+message Fcds{
+  string fcds_name = 1 [(validate.rules).string.min_len = 1];
```

Comment:
Every listener filter chain has a name so one can think of a listener as a collection of filter chains. Is this name referring to the collection name?

Reply:
That's right @kyessenov. It is referring to the name of the collection of listener filter chains under a listener. It is not used in the code anywhere today though. It's just a name to refer to.

Can I have multiple of those such collections of listener filter-chains under the same listener? No.

Reply:
Thanks for explaining. If two collections share a filter chain, then at the transport level, do we still have to get duplicate resources `<fcds_name/filter_chain.name>` if two listeners share a filter chain? I wonder if there is a way to have a global subscription instead of a per-listener subscription model.

Reply:
> Thanks for explaining. If two collections share a filter chain, then at the transport level, do we still have to get duplicate resources `<fcds_name/filter_chain.name>` if two listeners share a filter chain? I wonder if there is a way to have a global subscription instead of a per-listener subscription model.

Somewhat dependent on the use-case, but if you think of ADS, then there should be a single subscription to the management server, and it will notify the multiple "watchers".